hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/states/endpoint/VersionEndpointTask.java

```diff
@@ -88,11 +87,6 @@ public EndpointStateMachine.EndPointStates call() throws Exception {
           // Check HddsVolumes
           checkVolumeSet(ozoneContainer.getVolumeSet(), scmId, clusterId);
 
-          DatanodeLayoutStorage layoutStorage
-              = new DatanodeLayoutStorage(configuration);
-          layoutStorage.setClusterId(clusterId);
-          layoutStorage.persistCurrentState();
-
```

Comment:
Good catch on the bug!  There are multiple SCM endpoints so the  `persistCurrentState()` is called multiple times for the same file.